### PR TITLE
Make sure all contents in readme is included for secret management module

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

File: docs/src/main/asciidoc/legal.adoc [hunk 1]
```diff
@@ -1 +1,4 @@
+
+(C) 2016-2021 the original authors.
```
Comment (backwind1233) [inline docs/src/main/asciidoc/legal.adoc:2]: 2016-2021 --> 2016-2022?

Comment (backwind1233) [reply docs/src/main/asciidoc/legal.adoc:2]: what's this ?

**Review by backwind1233 — APPROVED** [review]
LGTM

File: docs/src/main/asciidoc/legal.adoc (hunk 1 above)
Comment (chenrujun) [reply docs/src/main/asciidoc/legal.adoc:2]: https://docs.spring.io/spring-framework/docs/current/reference/html/

image

**Pull request merged**